### PR TITLE
admin subscription view page with invoices

### PR DESCRIPTION
**Pull request by sonyy-m**

### CODE REVIEW

File: view/adminhtml/templates/subscription_view.phtml [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+$subscriptionInfo = $block->getSubscrib();
+$invoices = $block->getSubscriptionInvoice();
+?>
+<div class="page-main-actions">
+    <div class="page-actions" data-ui-id="page-actions-toolbar-content-header">
+        <div class="page-actions-inner" data-title="">
+            <div class="page-actions-buttons">
+                <button id="back" title="Back" type="button" class="action-default scalable back" onclick="location.href = '<?php echo $block->getUrl('*'); ?>'">
+                    <span>Back</span>
+                </button>
```
Comment (sofiya02) [inline view/adminhtml/templates/subscription_view.phtml:13]: add pause resume cancel action next to back button with logic (if active show pause and cancel and not show resume and cancel)

**Review by sofiya02 — COMMENTED** [review]
Change the following 
1. Subscription List - change to Subscriptions (text change)
2. on list page add view link to Subscription Id

File: view/adminhtml/templates/subscription_view.phtml (hunk 1 above)
Comment (sonyy-m) [reply view/adminhtml/templates/subscription_view.phtml:13]: already added based on subscription status only

**Review by sofiya02 — APPROVED** [review]
LGTM

**Pull request merged**